staging/src/k8s.io/apiserver/pkg/authorization/union/union.go

```diff
@@ -44,18 +44,18 @@ func New(authorizationHandlers ...authorizer.Authorizer) authorizer.Authorizer {
 // Authorizes against a chain of authorizer.Authorizer objects and returns nil if successful and returns error if unsuccessful
 func (authzHandler unionAuthzHandler) Authorize(ctx context.Context, a authorizer.Attributes) (authorizer.Decision, string, error) {
 	var (
-		errlist    []error
-		reasonlist []string
+		errList    []error
+		reasonList []string
```

Comment:
Thanks for the PR, but renaming local private variables like this is change without much benefit. I'd suggest reverting this file and limiting this PR to the typo fix.